HackingWithSwift/week2/day_12/Force unwrapping/Force unwrapping .playground/Contents.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+let str = "5"
+let num = Int(str)!
+
+let age = "24"
+let myAge = Int(age)!
+
+let salary = 100
+let strSalary = String(salary)
+
+let url = URL(string: "https://chatgpt.com")!
+
```

Comment:
I understand the goal of this exercise was to practice force unwrapping, but I’d encourage you to try handling it differently. Consider using `if let` or `guard let` statements and simply printing the unwrapped values. It’s just an extra exercise to explore different approaches.